--- uboot.mk.bak	2011-10-19 19:04:41.766483781 +0200
+++ uboot.mk	2011-10-19 19:42:17.926483134 +0200
@@ -75,6 +75,7 @@ define UBOOT_CONFIGURE_CMDS
 	$(call insert_define,CONFIG_SERVERIP,$(BR2_TARGET_UBOOT_SERVERIP))
 	$(call insert_define,CONFIG_ETHADDR,$(BR2_TARGET_UBOOT_ETHADDR))
 	$(call insert_define,CONFIG_ETH1ADDR,$(BR2_TARGET_UBOOT_ETH1ADDR))
+	@echo '#define CONFIG_BOOTCOMMAND "nboot.e kernel;bootm"' >> $(@D)/include/config.h
 	@echo "#endif /* __BR2_ADDED_CONFIG_H */" >> $(@D)/include/config.h
 endef
 
